inst/examples/index.Rmd

```diff
@@ -84,8 +84,51 @@ Note that this overriding mechanism also works for other output formats in **pag
 
 ## Print to PDF
 
-You can print HTML pages to PDF using Google Chrome or Chromium using the menu "Print" or by pressing `Ctrl + P` (or `Command + P` on macOS). Remember to allow background graphics to be printed. You may also consider using the function `pagedown::chrome_print()`, but currently it is experimental and you may get an incomplete PDF, so we recommend that you open the page in a real browser and print to PDF there instead.
+There are three ways to print to PDF:
 
+1. with Google Chrome or Chromium using the menu "Print" or by pressing `Ctrl + P` (or `Command + P` on macOS). Remember to allow background graphics to be printed.
+
+1. using the function `pagedown::chrome_print()`. Its first argument (`input`) accepts a path to a local Rmd or HTML file or an URL. Google Chrome or Chromium must be installed on your system.
+
+1. in RStudio, adding the following line in the YAML header of your Rmd file:
+
+   ```yaml
+   knit: pagedown::chrome_print
+   ```
+   
+   With this metadata parameter, the behavior of the "Knit" button of RStudio is modified: it produces both the HTML document and the PDF with Chrome. This functionality is suitable for any R Markdown HTML output format and is mainly convenient for small documents like presentations or notes.  
+   If `chrome_print()` cannot find Chrome or Chromium automatically, set the `PAGEDOWN_CHROME` environment variable to the path to Chrome or Chromium executable. 
+
+### Print to PDF on a server
+
+If you want to use `chrome_print()` or `knit_pdf_chrome()` on a server (an RStudio server, for instance), Chromium or Chrome has to be present on this server and available from the `PATH` or `PAGEDOWN_CHROME` environment variables.  
```

Comment:
`knit_pdf_chrome` is no more existing right ? 
I think it is a leftover from your first propsal.

Reply:
Good catch! 🕵

Reply:
```suggestion
If you want to use `chrome_print()` on a server (an RStudio server, for instance), Chromium or Chrome has to be present on this server and available from the `PATH` or `PAGEDOWN_CHROME` environment variables.  
```